Buildings/Controls/OBC/FDE/DOAS/Validation/DehumMode.mo

```diff
@@ -0,0 +1,45 @@
+within Buildings.Controls.OBC.FDE.DOAS.Validation;
```

Comment:
All the validation models should also have the info section. Also, a script should be created to run the unit tests `Simulate and Plot`.